tests/tracer/telemetry/test_writer.py

```diff
@@ -0,0 +1,142 @@
+import mock
+import pytest
+
+from ddtrace.internal.telemetry.writer import TelemetryWriter
+
+
+class FakeResponse:
+    def __init__(self, status):
+        self.status = status
+
+
+@pytest.fixture
+def mock_send_request_200():
+    with mock.patch.object(TelemetryWriter, "_send_request", return_value=FakeResponse(200)):
+        yield
```

Comment:
TODO: Look at HTTPretty for mocking http calls and asserting the values you wanted.